api/app/controllers/spree/api/base_controller.rb

```diff
@@ -106,7 +125,8 @@ def requires_authentication?
       end
 
       def not_found
-        render "spree/api/errors/not_found", status: 404 and return
+        render json: { error: I18n.t(:resource_not_found, scope: "spree.api") },
+          status: 404 and return
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use `&&` instead of `and`.